### PR TITLE
fix: remove hardcoded texts

### LINKED ISSUE #5210: [Bug]: Translations are missing
### Preflight Checklist

- [X] I could not find a solution in the documentation, the existing issues or discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Environment

Self-hosted

### Describe the bug

I found some hard coded labels. Maybe there are more.

### To reproduce

Go to Branding > Colors

### Screenshots

image

### Expected behavior

_No response_

### Version

_No response_

### Operating System

_No response_

### Relevant Configuration

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by doncicuto**
Reported hard coded texts have been replaced. I've spent a couple of hours checking for more hard coded texts but I couldn't find them. 

I've also reviewed some italian and german translations and proposed new translations. I've used google translate so I hope they're good to go. I'll change any wrong translation after your review.

Closes #5210

```[tasklist]
### Definition of Ready
- [X] I am happy with the code
- [X] Short description of the feature/issue is added in the pr description
- [X] PR is linked to the corresponding user story
- [X] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [X] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

**Comment by hifabienne** [comment]
@doncicuto Thanks for the contibution.
@peintnermax can you have a look at it?

File: console/src/assets/i18n/de.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     },
     "QUICKSTARTS": {
       "LABEL": "Erste Schritte",
-      "TITLE": "Quickstarts",
+      "TITLE": "Schnellstarts",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:34]:
```suggestion
      "TITLE": "Quickstarts",
```

File: console/src/assets/i18n/de.json [hunk 2]
```diff
@@ -78,17 +78,17 @@
...
+    "INSTANCE": "Instanz",
+    "DASHBOARD": "Startseite",
     "PERSONAL_INFO": "Persönliche Informationen",
     "DOCUMENTATION": "Dokumentation",
     "INSTANCEOVERVIEW": "Instanz",
     "ORGS": "Organisationen",
-    "VIEWS": "Views",
-    "EVENTS": "Events",
-    "FAILEDEVENTS": "Failed Events",
+    "VIEWS": "Ansichten",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:87]:
```suggestion
    "VIEWS": "Views",
```

File: console/src/assets/i18n/de.json [hunk 3]
```diff
@@ -78,17 +78,17 @@
...
+    "EVENTS": "Veranstaltungen",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:88]:
```suggestion
    "EVENTS": "Events",
```

File: console/src/assets/i18n/de.json [hunk 4]
```diff
@@ -78,17 +78,17 @@
...
+    "FAILEDEVENTS": "Gescheiterte Veranstaltungen",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:89]:
```suggestion
    "FAILEDEVENTS": "Fehlerhafte Events",
```

File: console/src/assets/i18n/de.json [hunk 5]
```diff
@@ -248,13 +248,13 @@
...
       "GENERAL": "Allgemein",
-      "IDP": "Identity Provider",
+      "IDP": "Identitätsanbieter",
       "SECURITY": "Passwort und Sicherheit",
       "KEYS": "Schlüssel",
-      "PAT": "Personal Access Tokens",
-      "USERGRANTS": "Authorizations",
-      "MEMBERSHIPS": "Memberships",
-      "METADATA": "Metadata"
+      "PAT": "Persönliche Zugriffstoken",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:254]:
```suggestion
      "PAT": "Personal Access Tokens",
```

File: console/src/assets/i18n/de.json [hunk 6]
```diff
@@ -248,13 +248,13 @@
...
+      "USERGRANTS": "Berechtigungen",
+      "MEMBERSHIPS": "Mitgliedschaften",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:256]:
```suggestion
      "MEMBERSHIPS": "Memberships",
```

File: console/src/assets/i18n/de.json [hunk 7]
```diff
@@ -315,7 +315,7 @@
       "CREATIONDATE": "Erstelldatum",
       "TYPES": {
         "HUMAN": "Benutzer",
-        "MACHINE": "Service Users"
+        "MACHINE": "Dienstbenutzer"
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:318]:
```suggestion
        "MACHINE": "Service User"
```

File: console/src/assets/i18n/de.json [hunk 8]
```diff
@@ -673,18 +673,18 @@
         "CHOOSEDATEAFTER": "Geben Sie ein valides Ablaufdatum an. Ab"
       },
       "ADDED": {
-        "TITLE": "Personal Access Token",
+        "TITLE": "Persönliches Zugriffstoken",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:676]:
```suggestion
        "TITLE": "Personal Access Token",
```

File: console/src/assets/i18n/de.json [hunk 9]
```diff
@@ -673,18 +673,18 @@
...
         "DESCRIPTION": "Kopieren Sie Ihr Access Token. Sie werden später nicht mehr darauf zugreifen können."
       },
       "DELETE": {
         "TITLE": "Token löschen",
         "DESCRIPTION": "Sie sind im Begriff das Token unwiderruflich zu löschen. Wollen Sie dies wirklich tun?"
       },
-      "DELETED": "Personal Access Token gelöscht."
+      "DELETED": "Persönliches Zugriffstoken gelöscht."
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:683]:
```suggestion
      "DELETED": "Personal Access Token gelöscht."
```

File: console/src/assets/i18n/de.json [hunk 10]
```diff
@@ -774,7 +774,7 @@
       "4": "Gestoppt"
     },
     "VIEWS": {
-      "TITLE": "Views",
+      "TITLE": "Ansichten",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:777]:
```suggestion
      "TITLE": "Views",
```

File: console/src/assets/i18n/de.json [hunk 11]
```diff
@@ -803,7 +803,7 @@
       "DELETESUCCESS": "Gescheiterte Events entfernt."
     },
     "EVENTS": {
-      "TITLE": "Events",
+      "TITLE": "Veranstaltungen",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:806]:
```suggestion
      "TITLE": "Events",
```

File: console/src/assets/i18n/de.json [hunk 12]
```diff
@@ -849,7 +849,7 @@
         "OTHERS": "weitere"
       },
       "DIALOG": {
-        "TITLE": "Event Detail"
+        "TITLE": "Ereignisdetail"
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:852]:
```suggestion
        "TITLE": "Eventdetail"
```

File: console/src/assets/i18n/de.json [hunk 13]
```diff
@@ -1926,10 +1930,10 @@
         "DESCRIPTION": "SAML Applikationen"
       },
       "CONFIGSECTION": "SAML Konfiguration",
-      "URL": "Url des Metadata Files",
+      "URL": "Url des Metadaten Files",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:1933]:
```suggestion
      "URL": "Url der Metadaten-Datei",
```

File: console/src/assets/i18n/it.json [hunk 14]
```diff
@@ -9,8 +9,8 @@
   "FOOTER": {
     "LINKS": {
       "CONTACT": "Kontakt",
-      "TOS": "Terms of Service",
-      "PP": "Privacy Policy"
+      "TOS": "Termini di servizio",
+      "PP": "Politica sulla riservatezza"
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:13]:
```suggestion
      "PP": "Informativa sulla privacy"
```

File: console/src/assets/i18n/it.json [hunk 15]
```diff
@@ -416,7 +416,7 @@
       "EMPTY": "Nessun IDP esterno trovato",
       "DIALOG": {
         "DELETE_TITLE": "Rimuovere l'IDP",
-        "DELETE_DESCRIPTION": "Stai per eliminare un Identity Provider da un utente. Vuoi continuare?"
+        "DELETE_DESCRIPTION": "Stai per eliminare un Fornitore di identità da un utente. Vuoi continuare?"
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:419]:
```suggestion
        "DELETE_DESCRIPTION": "Stai per eliminare un fornitore di identità da un utente. Vuoi continuare?"
```

File: console/src/assets/i18n/it.json [hunk 16]
```diff
@@ -992,10 +992,10 @@
       "NOTIFICATIONS": "Impostazioni di notifica",
       "NOTIFICATIONS_DESC": "Impostazioni SMTP e SMS",
       "MESSAGETEXTS": "Testi di notifica",
-      "IDP": "Identity Providers",
+      "IDP": "Fornitori di identità",
       "DOMAIN": "Impostazioni del dominio",
       "LOGINTEXTS": "Testi dell'interfaccia login",
-      "BRANDING": "Branding",
+      "BRANDING": "Marchio",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:998]:
```suggestion
      "BRANDING": "Branding",
```

File: console/src/assets/i18n/it.json [hunk 17]
```diff
@@ -1120,7 +1120,7 @@
       "PASSWORDCHANGE": "Cambiamento della password"
     },
     "PRIVATELABELING": {
-      "TITLE": "Branding",
+      "TITLE": "Marchio",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1123]:
```suggestion
      "TITLE": "Branding",
```

File: console/src/assets/i18n/it.json [hunk 18]
```diff
@@ -1143,6 +1143,10 @@
       "MAXSIZEEXCEEDED": "Dimensione massima di 524kB superata.",
       "NOSVGSUPPORTED": "SVG non sono supportati",
       "FONTINLOGINONLY": "Il carattere \u00e8 attualmente visualizzato solo nell'interfaccia di accesso.",
+      "BACKGROUNDCOLOR": "Colore di sfondo",
+      "PRIMARYCOLOR": "Colore primario",
+      "WARNCOLOR": "Colore di aviso",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1148]:
```suggestion
      "WARNCOLOR": "Colore di avviso",
```

File: console/src/assets/i18n/it.json [hunk 19]
```diff
@@ -1143,6 +1143,10 @@
...
+      "FONTCOLOR": "Colore del carattere",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1149]:
```suggestion
      "FONTCOLOR": "Colore del testo",
```

File: console/src/assets/i18n/it.json [hunk 20]
```diff
@@ -1167,14 +1171,14 @@
       "TITLE": "Impostazioni dominio"
     },
     "PRIVATELABELING_POLICY": {
-      "TITLE": "Branding",
+      "TITLE": "Marchio",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1174]:
```suggestion
      "TITLE": "Branding",
```

File: console/src/assets/i18n/it.json [hunk 21]
```diff
@@ -1167,14 +1171,14 @@
...
       "BTN": "Seleziona File",
       "DESCRIPTION": "Personalizza l'aspetto del Login",
       "ACTIVATEPREVIEW": "Attiva la configurazione"
     },
     "LOGIN_POLICY": {
       "TITLE": "Impostazioni di accesso",
-      "DESCRIPTION": "Definisci come gli utenti possono essere autenticati e configura gli Identity Provider",
+      "DESCRIPTION": "Definisci come gli utenti possono essere autenticati e configura gli Fornitore di identità",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1181]:
```suggestion
      "DESCRIPTION": "Definisci come gli utenti possono essere autenticati e configura i fornitori di identità",
```

File: console/src/assets/i18n/it.json [hunk 22]
```diff
@@ -1393,21 +1397,21 @@
         "GRANTED_SINGULAR": "Progetto delegato"
       },
       "PRIVATELABEL": {
-        "TITLE": "Impostazione Branding",
+        "TITLE": "Impostazione marchio",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1400]:
```suggestion
        "TITLE": "Impostazione branding",
```

File: console/src/assets/i18n/it.json [hunk 23]
```diff
@@ -1393,21 +1397,21 @@
...
         "0": {
           "TITLE": "Non specificato",
-          "DESC": "Non appena l'utente viene identificato, viene mostrata l'impostazione Branding dell'organizzazione, prima che venga mostrato il default del sistema."
+          "DESC": "Non appena l'utente viene identificato, viene mostrata l'impostazione marchio dell'organizzazione, prima che venga mostrato il default del sistema."
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1403]:
```suggestion
          "DESC": "Non appena l'utente viene identificato, viene mostrata l'impostazione branding dell'organizzazione, prima che venga mostrato il default del sistema."
```

File: console/src/assets/i18n/it.json [hunk 24]
```diff
@@ -1393,21 +1397,21 @@
...
         },
         "1": {
           "TITLE": "Applica l'impostazione del progetto",
-          "DESC": "Branding dell'organizzazione del progetto sar\u00e0 mostrata"
+          "DESC": "Marchio dell'organizzazione del progetto sar\u00e0 mostrata"
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1407]:
```suggestion
          "DESC": "Branding dell'organizzazione del progetto sar\u00e0 mostrata"
```

File: console/src/assets/i18n/it.json [hunk 25]
```diff
@@ -1393,21 +1397,21 @@
...
         },
         "2": {
           "TITLE": "Applica l'impostazione dell' organizzazione dell' utente",
-          "DESC": "Verr\u00e0 mostrata l'impostazione Branding dell'organizzazione del progetto, ma non appena l'utente viene identificato, verr\u00e0 mostrata l'impostazione dell'organizzazione dell'utente identificato."
+          "DESC": "Verr\u00e0 mostrata l'impostazione marchio dell'organizzazione del progetto, ma non appena l'utente viene identificato, verr\u00e0 mostrata l'impostazione dell'organizzazione dell'utente identificato."
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1411]:
```suggestion
          "DESC": "Verr\u00e0 mostrata l'impostazione branding dell'organizzazione del progetto, ma non appena l'utente viene identificato, verr\u00e0 mostrata l'impostazione dell'organizzazione dell'utente identificato."
```

File: console/src/assets/i18n/it.json [hunk 26]
```diff
@@ -1393,21 +1397,21 @@
...
         },
         "DIALOG": {
-          "TITLE": "Impostazione Branding",
+          "TITLE": "Impostazione marchio",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1414]:
```suggestion
          "TITLE": "Impostazione branding",
```

File: console/src/assets/i18n/it.json [hunk 27]
```diff
@@ -1610,16 +1614,16 @@
   },
   "IDP": {
     "LIST": {
-      "TITLE": "Identity Providers",
-      "DESCRIPTION": "Definisci ulteriori Identity Provider, che possono essere utilizzati per l'autenticazione nelle tue organizzazioni.",
-      "ACTIVETITLE": "Identity Providers"
+      "TITLE": "Fornitori di identità",
+      "DESCRIPTION": "Definisci ulteriori Fornitore di identità, che possono essere utilizzati per l'autenticazione nelle tue organizzazioni.",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1618]:
```suggestion
      "DESCRIPTION": "Definisci ulteriori fornitori di identità, che possono essere utilizzati per l'autenticazione nelle tue organizzazioni.",
```

File: console/src/assets/i18n/it.json [hunk 28]
```diff
@@ -1647,7 +1651,7 @@
       "0": "No Styling",
       "1": "Google"
     },
-    "ADD": "Aggiungi Identity Provider",
+    "ADD": "Aggiungi Fornitore di identità",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:1654]:
```suggestion
    "ADD": "Aggiungi fornitore di identità",
```

**Review by peintnermax — COMMENTED** [review]
I think its ok to use the english terms in some use cases. For example it does not make sense to translate `Events` and `Views` as these are strongly bound to our eventstore and documentation. Same for `Personal Access Token`  @eliobischof do your have other opinions?

File: console/src/assets/i18n/de.json (hunk 6 above)
Comment (eliobischof) [reply console/src/assets/i18n/de.json:256]: hm, this is a hard one

Comment (eliobischof) [reply console/src/assets/i18n/de.json:256]:
```suggestion
      "MEMBERSHIPS": "Manager-Rollen",
```

This is consistent with the title
image

**Comment by eliobischof** [comment]
> I think its ok to use the english terms in some use cases. For example it does not make sense to translate `Events` and `Views` as these are strongly bound to our eventstore and documentation. Same for `Personal Access Token` @eliobischof do your have other opinions?

I second that.

**Comment by doncicuto** [comment]
Thanks for your review @peintnermax and @eliobischof, as stated earlier I use Google Translate so I apologize for incoherent texts, hope that this PR is still useful after all

**Comment by peintnermax** [comment]
of course its useful.. LGTM right now.. there are just some conflicts 👍

**Comment by hifabienne** [comment]
@doncicuto can you have a look at the conflicts?

**Comment by doncicuto** [comment]
Hi @peintnermax and @adlerhurst I've fixed pending conflicts, hope that PR is ready to merge

**Review by peintnermax — APPROVED** [review]
thx for your work 👍

**Pull request merged**